latitudesh/lsh
# Chore: clear codebase

#### What does this PR do?
Remove Unused and/or Unwanted code and comments

#### How to test
- Build the CLI by running `go build -o lsh cmd/lsh/main.go` in the project folder.
- Verify if everything is working fine
